src/index.js

```diff
@@ -94,6 +94,15 @@ window.App = {
     const myNametag = document.querySelector("#player-rig .nametag");
     myNametag.setAttribute("text", "value", username);
 
+    //change spawn position & rotation
+    const myHead = document.querySelector("#player-rig #head");
```

Comment:
This stuff should definitely live inside a component that we just put on the player rig, see https://github.com/networked-aframe/networked-aframe/blob/master/server/static/js/spawn-in-circle.component.js for an example that I think pretty much does this same thing. We may want to just use that but going to comment on the rest of the PR anyway just for learning purposes.